skbio/io/tests/test_clustal.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
Actually, the shebang line is useful in any file with a `if __name__ = '__main__'`. It's a matter of taste but it allows running them with `$ foo.py` instead of `$ python foo.py`.